content/en/docs/reference/config-api/kubeadm-config.v1beta3.md

```diff
@@ -1413,4 +1413,117 @@ first alpha-numerically.</td>
 </tbody>
 </table>
     
+
+## `BootstrapToken`     {#BootstrapToken}
```

Comment:
this is fine like this, but i don't think this scales well longterm.
technically BootstrapToken is no longer a kubeadm "config" API type. it's a separate group.
instead of generating it here you could link to godoc and treat it as external.

also IMO, this page:
https://kubernetes.io/docs/reference/config-api/

should group APIs by components:
kube-proxy APIs, kubeadm APIs, client API, kube-apiserver APIs.

if an API is shared we could list it at multiple places.

kubeadm has:
- kubeadm.k8s.io (main config API)
- output.kubeadm.k8s.io (alpha, machine readable output support)
- bootstraptoken.kubeadm.k8s.io (used by output and main config)

kube-apiserver has:
...

Reply:
In the long run, I'd like to see kubadm config code migrated into staging.

Reply:
Moving the kubeadm code to a separate repository will likely happen after kubectl moves out completely. Staging has limbo problems.

That would not solve the organizational improvement mentioned above.